report_thesis/src/sections/optuna_ranges.tex

```diff
@@ -0,0 +1,101 @@
+\toprule
+\textbf{Model}                       & \textbf{Parameter}                & \textbf{Range}                           \\ \midrule
+\multirow{2}{*}{PCA}                 & n\_components                     & 1 - 50                                   \\ \cmidrule{2-3}
+                                     & whiten                            & \{True, False\}                          \\ \midrule
+\multirow{4}{*}{KernelPCA}           & n\_components                     & 1 - 100                                  \\ \cmidrule{2-3}
+                                     & kernel                            & \{linear, poly, rbf, sigmoid, cosine\}   \\ \cmidrule{2-3}
+                                     & gamma                             & $10^{-3}$ - $10^{1}$ (log scale)         \\ \cmidrule{2-3}
+                                     & degree                            & 1 - 5                                    \\ \midrule
+\multirow{2}{*}{RobustScaler}        & quantile\_range                   & \{25-75, 10-90, 5-95, 35-65, 30-70, 40-60\} \\ \cmidrule{2-3}
+                                     & with\_centering                   & \{True, False\}                          \\ \midrule
+\multirow{2}{*}{StandardScaler}      & with\_mean                        & \{True, False\}                          \\ \cmidrule{2-3}
+                                     & with\_std                         & \{True, False\}                          \\ \midrule
+MinMaxScaler                         & feature\_range                    & \{0,1\}, \{-1,1\}                        \\ \midrule
+\multirow{2}{*}{PowerTransformer}    & method                            & yeo-johnson                              \\ \cmidrule{2-3}
+                                     & standardize                       & \{True, False\}                          \\ \midrule
+\multirow{3}{*}{QuantileTransformer} & n\_quantiles                      & 100 - 1000                               \\ \cmidrule{2-3}
+                                     & output\_distribution              & \{uniform, normal\}                      \\ \cmidrule{2-3}
+                                     & subsample                         & 10000 - 100000                           \\ \midrule
+MaxAbsScaler                         & -                                 & -                                        \\ \midrule
+Norm3Scaler                          & -                                 & -                                        \\ \midrule
+\end{tabular}
+\label{tab:optuna_model_configurations}
+\caption{Optuna preprocessing configuration ranges.}
+\end{table*}
+
+
+For the models, we include a range of parameters for various regressors, as shown in Table~\ref{tab:optuna_model_configurations}.
+Similar to the preprocessing parameters, we aim to cover a wide range of regressors and their respective hyperparameters while maintaining a balance between coverage and computational efficiency.
```

Comment:
> Also, I tried to emphasize that the chosen parameter ranges are not exhaustive, and that we have to consider how much time it takes to run as well. Does this point come across clearly for both the preprocessing and model configurations?

yes the point is being made, but i'd be a bit more careful using `computational efficiency` when referring to not having infinite time to run infinite runs